inference-engine/thirdparty/clDNN/src/program.cpp

```diff
@@ -126,7 +128,9 @@ program_impl::~program_impl() {
 void program_impl::init_primitives() {
     static bool is_initialized = false;
     if (!is_initialized) {
-        gpu::register_implementations_gpu();
+        common::register_implementations_common();
+        cpu::register_implementations_cpu();
+        ocl::register_implementations_ocl();
```

Comment:
Very minor but it is already in the namespace so I think there is no need to _[imple_name]?

Reply:
fixed